src/stores/search.ts

```diff
@@ -152,6 +153,44 @@ export const useSearchStore = defineStore('search', {
     },
   },
   actions: {
+    /**
+     * Updates the search type and search term, and returns the
+     * updated localized search path.
+     */
+    updateSearchPath({
+      type,
+      searchTerm,
+    }: { type?: SearchType; searchTerm?: string } = {}): string {
+      if (type) {
+        this.setSearchType(type)
+      }
+      if (searchTerm) {
+        this.setSearchTerm(searchTerm)
+      }
+
+      return this.$nuxt.localePath({
+        path: searchPath(this.searchType),
+        query: this.searchQueryParams as Dictionary<string>,
+      })
```

Comment:
Could this call `getSearchPath` internally instead, to prevent drift between the two?

Reply:
Thank you for this suggestion! I was thinking about doing so, but didn't have a compelling enough reason :)